etna/records/views/records.py

```diff
@@ -37,13 +37,13 @@ def record_disambiguation_view(request, reference_number):
         return record_detail_view(request, record.iaid)
 
     paginator = APIPaginator(count, per_page=per_page)
-    page = Page(records, number=page_number, paginator=paginator)
+    record_pages = Page(records, number=page_number, paginator=paginator)
 
     return render(
         request,
         "records/record_disambiguation_page.html",
         {
-            "pages": page,
+            "record_pages": record_pages,
             "queried_reference_number": reference_number,
```

Comment:
@TNA-Allan I don't think `page` is necessarily a bad name here, as it represents a "page of results" from the API. If we don't think that's descriptive enough, perhaps `paginator_page` would make things clearer?

Reply:
It isn't the `paginator` itself though... it is a 'page' of results (extracted from the `paginator`). Other okay names might be:
- `current_page`
- `record_results_page`

Reply:
Thanks for clarifying. Going with `record_results_page`